exe/rubydown

```diff
@@ -62,6 +62,9 @@ class EvalDoc < Kramdown::Document
     case
     when eval_result.respond_to?(:to_html)
       Kramdown::Document.new(eval_result.to_html, input: :html).root
+    when eval_result.instance_of?(File) && eval_result.path[-4,4] == ".png"
```

Comment:
I prefer `path.end_with?('.png')` to `path[-4,4] == ".png"`.

Reply:
Thanks, but I will use
```
File.extname(eval_result.path) == ".png"
```
instead.